sympy/physics/vector/vector.py

```diff
@@ -345,12 +339,20 @@ def __ror__(self, other):
     def __rsub__(self, other):
         return (-1 * self) + other
 
-    def __str__(self, printer=None):
+    def __str__(self, printer=None, order=True):
```

Comment:
@isuruf I thought SymPy gets around this by ordering the dictionaries before printing, if you use a Printer.